sdk/script/management-sdk.ts

```diff
@@ -193,11 +185,6 @@ export class AccountManager {
             .then((res: JsonResponse) => res.body.deployment);
     }
 
-    public getDeploymentMetrics(appName: string, deploymentName: string): Promise<DeploymentMetrics> {
-        return this.get(urlEncode `/apps/${appName}/deployments/${deploymentName}/metrics`)
-            .then((res: JsonResponse) => res.body.metrics);
-    }
-
     public updateDeployment(appName: string, deploymentName: string, infoToChange: Deployment): Promise<void> {
```

Comment:
Should we expose "renameDeployment" and "renameApp" methods to better reflect the CLI and the general purpose of editing either entity type? If this method is needed by the CLI, maybe we could just not document it, and create a rename variant to simply "wraps" the more general-purpose update methods.